OKTA-736868 - Rate Limits - Update DynamicScale endpoints (#4868)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/rl-dashboard/index.md

Each API is a link that navigates you to the [dashboard view of API rate limits and violations](#api-rate-limits-and-violations) where you can obtain more details on the API.

> You can perform a search in your table list by making an entry in the Search box. You can also perform a "lookup" with the search. For example, you're calling Okta with an example `requestURI` and you want to know the rate limit that is applied to calls that are made against that `requestURI`.

Author: Search is not really a UI element unless you consider the hint text within the field. I'll keep it lower case and remove box, i.e.: `Search box` is now `search field`.

You can enable per-client rate limiting to prevent a single client from blocking Internet traffic to your org's authentication and access APIs. See [Client-based rate limits](/docs/reference/rl-clientbased/). For these APIs, select from the following options:

> * **Enforce and log per client (recommended):** enables client-based rate limiting for all clients (default setting)

Author: acrolinx wanted these lower case, but I prefer the capitalization. Hopefully, we won't be penalized on it.

* **Log per client:** enables client-based rate limiting in log-only mode. Rate limiting is based on org-wide rate-limit values, but the client-specific rate limiting information is recorded as System Log events. By analyzing these events, you can determine if client-based rate limiting is effective for you.

> * **Do nothing (not recommended):** disables client-based rate limiting

Reviewer: ```suggestion
* **Do nothing (not recommended):** Disables client-based rate limiting
```

## API rate limits and violations

> The dashboard enables you to view an API, the current rate limit for each API, and usage of the API over a time period. You can select the API of interest and get more details such as the API's average usage and the top 10 IPs that are consuming the most traffic for the API.

Reviewer: ```suggestion
The dashboard enables you to view an API, the current rate limit for each API, and use of the API over a time period. You can select the API of interest and get more details such as the API's average use and the top 10 IPs that are consuming the most traffic for the API.
```

The line graph can show the number of unique IPs that are creating traffic at a given minute. For example, the line graph can show that many users are signing in at the same time, perhaps to attend a company-wide meeting. By comparing the number of unique IPs to the number of unique requests, you can see almost the same number of IPs as the number of requests. This means that there are many IPs creating the traffic and not just one IP consuming all the traffic.

#### Combination of the line graph and timeline graph view

The timeline graph gives you an overview of use from the days in the past week so that you can drill in to other time periods from the previous week. You can select one day at a time within the past week, from the dropdown menu, and view endpoint use one hour at a time in the line graph. These views enable you to understand whether this rate limit was an abnormal spike in use compared to regular use up to that point in time.

#### Bar graph view

> The bar graph compares a rogue IP or App with other IPs or Apps. For example, the bar graph can show a single rogue IP that is responsible for a spike in traffic that appears in the line graph. It can also show that many users are signing in at the same time, perhaps to attend a company-wide meeting or to check out a special offer that was delivered in a mass email to a company's clients.

Reviewer: ```suggestion
The bar graph compares a rogue IP or app with other IPs or apps. For example, the bar graph can show a single rogue IP that's responsible for a spike in traffic that appears in the line graph. It can also show that many users are signing in at the same time, perhaps to attend a company-wide meeting or to check out a special offer that was delivered in a mass email to a company's clients.
```

### Potential next steps

> If you find that the rate limit usage is high, you may want to adjust your rate limit maximums. If your company is planning an upcoming event, such as an upcoming marketing campaign, you may want to figure out the rate limit capacity that you'd need. In both cases, Okta recommends that you contact Okta support to determine and adjust your rate limits, see [How to Request a Temporary Rate Limit Increase](https://support.okta.com/help/s/article/How-can-we-request-to-have-the-rate-limit-for-our-org-temporarily-increased?language=en_US).

Reviewer: ```suggestion
If you find that the rate limit use is high, you may want to adjust your rate limit maximums. If your company is planning an upcoming event, such as an upcoming marketing campaign, you may want to figure out the rate limit capacity that you'd need. In both cases, contact Okta support to determine and adjust your rate limits. See [How to Request a Temporary Rate Limit Increase](https://support.okta.com/help/s/article/How-can-we-request-to-have-the-rate-limit-for-our-org-temporarily-increased?language=en_US).
```
